test/test_group.py

```diff
@@ -140,3 +142,13 @@ def test_remove_user_missing_group_id(self):
         single_group._users = []
         self.assertRaises(TSC.MissingRequiredFieldError, self.server.groups.remove_user, single_group,
                           '5de011f8-5aa9-4d5b-b991-f462c8dd6bb7')
+
+    def test_create_group(self):
+        with open(CREATE_GROUP, 'rb') as f:
+            response_xml = f.read().decode('utf-8')
+        with requests_mock.mock() as m:
+            m.post(self.baseurl, text=response_xml)
+            group_to_create = TSC.GroupItem('test')
```

Comment:
Can I be high maintenance and request a non-ascii name in a test as well?

I just remember some bugs in early sample code and libs prototypes around this.